core/field_dropdown.js

```diff
@@ -531,6 +532,28 @@ Blockly.FieldDropdown.prototype.renderSelectedText_ = function() {
       Blockly.Field.X_PADDING;
 };
 
+/**
+ * @override
```

Comment:
So, according to closure compiler [docs](https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#inheritdoc), @``override`` is preferred. If you do an @``override`` without any other properties, the rest of the jsdoc is inherited. Here's my understanding of what the two tags mean:
- @``inheritDoc`` if you're intentionally hiding a method, or changing the interface (eg: number of parameters, type of parameters). @``inheritDoc`` also implicitly inherits the @``override`` tag.
- @``override`` (preferred) if implementing a method or property that is part of the superclass, or if you're changing the method or interface. 

So pretty much always use @``override``. if you dont have any other jsdoc, you'll inherit it, otherwise you can replace it with your own if changing types etc.